procrastinate/sql/schema.sql

```diff
@@ -59,6 +60,12 @@ CREATE TABLE procrastinate_events (
     at timestamp with time zone DEFAULT NOW() NULL
 );
 
+CREATE TABLE procrastinate_worker_heartbeats(
```

Comment:
What do you think of dropping the ˋ_heartbeat` suffix from that table?

This will make it possible to extend the table with columns not related to heartbeats.